postreise/plot/plot_utilization_map.py

```diff
@@ -35,92 +20,109 @@ def map_risk_bind(
+        If None, default to `postreise.plot.colors.traffic_palette`.
+    :param int/float all_branch_scale_factor: scale factor for unhighlighted branches.
+    :param int/float all_branch_min_width: minimum width for unhighlighted branches.
+    :param int/float select_branch_scale_factor: scale factor for highlighted branches.
+    :param int/float select_branch_min_width: minimum width for highlighted branches.
+    :param tuple(int, int) figsize: size of the bokeh figure (in pixels).
+    :param bool show_color_bar: whether to render the color bar on the figure.
+    :param dict plot_states_kwargs: keyword arguments to be passed to
+        :func:`postreise.plot.plot_states.plot_states`.
+    :return: (*bokeh.plotting.figure*) -- map of lines with risk and bind incidents
+        color coded.
     """
-    if us_states_dat is None:
-        us_states_dat = get_state_borders()
-
     if risk_or_bind == "risk":
         risk_or_bind_units = "Risk (MWH)"
 
     if risk_or_bind == "bind":
         risk_or_bind_units = "Binding incidents"
 
+    if palette is None:
+        palette = list(traffic_palette)
+
     # projection steps for mapping
     branch_congestion = pd.concat(
         [branch.loc[congestion_stats.index], congestion_stats], axis=1
     )
     branch_map_all = project_branch(branch)
-    multi_line_source_all = ColumnDataSource(
-        {
-            "xs": branch_map_all[["from_x", "to_x"]].values.tolist(),
-            "ys": branch_map_all[["from_y", "to_y"]].values.tolist(),
-            "cap": branch_map_all["rateA"] / 2000 + 0.2,
-        }
-    )
-    a, b = project_borders(us_states_dat)
-    tools: str = "pan,wheel_zoom,reset,save"
 
     branch_congestion = branch_congestion[branch_congestion[risk_or_bind] > 0]
     branch_congestion.sort_values(by=[risk_or_bind])
     branch_map = project_branch(branch_congestion)
     min_val = branch_congestion[risk_or_bind].min() if vmin is None else vmin
     max_val = branch_congestion[risk_or_bind].max() if vmax is None else vmax
     mapper = linear_cmap(
-        field_name=risk_or_bind, palette=traffic_palette, low=min_val, high=max_val
-    )
-    color_bar = ColorBar(
-        color_mapper=mapper["transform"],
-        width=385 if is_website else 500,
-        height=5,
-        location=(0, 0),
-        title=risk_or_bind_units,
-        orientation="horizontal",
-        padding=5,
+        field_name=risk_or_bind, palette=palette, low=min_val, high=max_val
     )
     multi_line_source = ColumnDataSource(
         {
             "xs": branch_map[["from_x", "to_x"]].values.tolist(),
             "ys": branch_map[["from_y", "to_y"]].values.tolist(),
             risk_or_bind: branch_map[risk_or_bind],
             "value": branch_map[risk_or_bind].round(),
-            "cap": branch_map["capacity"] / 1000 + 2,
+            "cap": (
+                branch_map["capacity"] * select_branch_scale_factor
+                + select_branch_min_width
+            ),
```

Comment:
the width of the line (in pixels) will be set by `"cap"`. From the user perspective, I am not sure the scaling is unnatural (default `select_branch_scale_factor=1e-3`). What do you think of taking the `log` of the capacity (always in MW) that will likely produce value in ]0, 2] or so and the user will be able to scale with factor around 1?

Reply:
The current scaling factor units are `pixels/MW`, which I think is nice because it's an easily-understood mapping. What if we changed it to `pixels/GW`, and added these units to the docstrings? That would produce default factors in the [0, 2] range.

Reply:
That sounds good. I think having a default scaling factor of 1 is nice.

Reply:
Done.